tests/e2e/coverage/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+# Start the main process in the background
+/manager --health-probe-bind-address=:8081 --metrics-bind-address=:8080 --leader-elect &
+export NUMAPLANE_PID=$!
+## Write NUMAPLANE_PID to a file
+echo $NUMAPLANE_PID > /numaplane.pid
+
+# Function to handle signals
+trap "echo 'Stopping PID $PID'; kill $PID; wait $PID; /manager --health-probe-bind-address=:8081 --metrics-bind-address=:8080 --leader-elect &" SIGTERM SIGINT
```

Comment:
So, if you kill this bash script, then it sends the signal to numaplane?

I guess the CI is sending the signal directly to numaplane's PID, right? Was this just another way for you to be able to test it?